### PR TITLE
Fixed Peacock setup doc and added doc for THM plugins

### PR DESCRIPTION
**Pull request by joshuahansel**
This PR:

- Changed doc to not instruct users to create a Peacock conda environment
- Simplified/shortened Peacock setup instructions
- Deleted unused doc files
- Added a THM utilities page giving an overview of its Peacock plugins

### CODE REVIEW

File: python/doc/content/python/peacock.md [hunk 1]
```diff
@@ -1,96 +1,49 @@
...
-```bash
-echo $0
--zsh
+export PATH=~/projects/moose/python/peacock:$PATH
 ```
 
-Where `-zsh` is our example result. The following table describes which profile file one must edit,
-as well as how to assign a value to a variable in that shell:
-
-| Shell | Profile Location | Command |
```
Comment (GiudGiud) [inline python/doc/content/python/peacock.md:42]: this was good dont delete it?

**Review by GiudGiud — APPROVED** [review]

**Comment by joshuahansel** [comment]
I hadn't noticed I messed up a link on the THM utilities, so I fixed that. Sorry about that.

**Comment by joshuahansel** [comment]
@GiudGiud Note I added a few additional commits for misc. doc improvements I had lying around.

**Review by GiudGiud — APPROVED** [review]
I think your last commit closes https://github.com/idaholab/moose/issues/28059

**Comment by joshuahansel** [comment]
> I think your last commit closes #28059

😆 Accurate issue name - didn't see that before, but I definitely agree. I'll update commit with the reference.

**Pull request merged**